spec/support/mock_api.rb

```diff
@@ -0,0 +1,48 @@
+module AnsibleTowerClient
```

Comment:
Do we want the `MockApi` available via the `AnsibleTowerClient` namespace? - It seems like maybe it should live under another namespace - possibly `AnsiblTowerClientTesting` or `AnsibleTowerClientMocks` - I guess in that case we could simply call the class `Api` without a collision.